src/components/dls/Switch/Switch.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+
+import styles from './Switch.module.scss';
+
+// reference: https://vercel.com/design/switch
+
+type Item = {
+  name: string;
+  value: string;
+  width: number;
```

Comment:
we shouldn't use `width`

Reply:
yeah, let's not make width a prop. We can have a `fullWidth` option that takes 100% of the container width.